### PR TITLE
Update README.adoc

### PR DESCRIPTION
**Pull request by gkwan-ibm**
add cloud-hosted guide instructions

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -60,6 +62,30 @@ You can access the following two microservices to test their availability:
...
+
+Run the following curl command to test the availability of the **inventory** microservice and 
+retrieve the information for a list of all previously registered hosts:
+```
+curl http://localhost:9080/inventory/systems
+```
+{: codeblock}
+
+In addition, you can run the following curl command to access a third microservice, 
+which retrieves and aggregates all of the configuration properties and sources that have been added throughout this guide.
```
Comment (dmuelle) [inline README.adoc:83]: ...all of the configuration properties and sources that have been added throughout this guide
--->
...all of the configuration properties and sources that are added throughout this guide

**Review by dmuelle — COMMENTED** [review]
looks good, one minor edit suggestion

File: README.adoc (hunk 1 above)
Comment (gkwan-ibm) [reply README.adoc]: updated both static and cloud-hosted instructions

**Review by dmuelle — APPROVED** [review]
LGTM

**Pull request merged**